tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
+
+[ -z "${KUBECONFIG}" ] && echo "KUBECONFIG is not set" && exit 1
+
+cat << EOF > test_config.json
+{
+ "storageClassLocal": "hostpath-provisioner",
+ "storageClassHostPath": "hostpath-provisioner",
+ "storageClassRhel": "hostpath-provisioner",
+ "storageClassWindows": "hostpath-provisioner",
+ "manageStorageClasses": false
+}
+EOF
+
+oc create namespace kubevirt
+rm -rf _out
+hack/dockerized hack/build-func-tests.sh
+make manifests
+rm -f _out/manifests/testing/cdi-*
+rm -f _out/manifests/testing/kubevirt-config.yaml
+sed -i "s/namespace: kubevirt/namespace: ${KUBEVIRTNAMESPACE}/g" _out/manifests/testing/*.yaml
+_out/tests/tests.test --cdi-namespace=${KUBEVIRTNAMESPACE} \
+	--config=${PWD}/test_config.json \
```

Comment:
@EdDev the --config flag also not mandatory because the code uses tests/default-config.json by default. If we want to give the user the ability to control storage configuration we need to parse it when the script is called.

Reply:
Done